Support URLRouter with include (#2110)
django/channels: docs/topics/routing.rst

    stream = self.scope["url_route"]["kwargs"]["stream"]

You can use [include](https://docs.djangoproject.com/en/5.1/ref/urls/#include) 
function for nested routings. This is similar as Django's URL routing system. 

Here's an example for nested routings. When you configure the routings in parent ``routings.py``;

.. code-block:: python

    urlpatterns = [

>         path("app1/", include("src.app1.routings"), name="app1"),

Reviewer: ```suggestion
        path("app1/", include("app1.routings"), name="app1"),
```

The `src` generally isn't on the import path.

Author: I removed it.

    ]

and in child ``app1/routings.py``;

.. code-block:: python

    app_name = 'app1'

    urlpatterns = [
        re_path(r"chats/(\d+)/$", test_app, name="chats"),
    ]

> you can establish the connection via the path such like ``/app1/chats/5/``.

Reviewer: ```suggestion
This would resolve to a path such as ``/app1/chats/5/``. 
```
